copy/multiple.go

```diff
@@ -124,8 +134,9 @@ func (c *copier) copyMultipleImages(ctx context.Context, policyContext *signatur
 	for i, instance := range instanceCopyList {
 		// Update instances to be edited by their `ListOperation` and
 		// populate necessary fields.
-		switch instance.op {
```

Comment:
Had to remove the `switch` cause I was `instanceCopyClone` to be processed in `instanceCopyCopy's block` as well as in its own block for replication.